kubernetes/samples/charts/util/setup.sh

```diff
@@ -21,11 +45,11 @@ function createVoyager() {
   fi
   echo
 
-  if [ "$(helm list | grep $VNAME |  wc -l)" = 0 ]; then
-    echo "Ihstall voyager operator."
+  if [ "$(helm list ${v_list_args} | grep $VNAME |  wc -l)" = 0 ]; then
+    echo "Install voyager operator."
     
-    helm install appscode/voyager --name $VNAME --version 7.4.0 \
-      --namespace voyager \
+    ${v_helm_install} --version 7.4.0 \
```

Comment:
probably a good idea to make the voyager version a variable

Reply:
We are suggesting this as sample on development environment not on production.  Certification Matrix will grow if we verify each version of Voyager which is not in our control. We have already big matrix of K8s Version/Helm Version/Prometheous Version

Reply:
Provide a new Optional Positional Parameter to take Custom Voyager Version
e.g.
 $ sh setup.sh
 Detected Helm Version [V3]
 [ERROR] Requires atleast 2 positional parameters
 usage: setup.sh create|delete traefik|voyager [voyager version]